### PR TITLE
[ZH] Cleanup `Gameplay/Hit object`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

## Changes

- Mostly formatting changes (ones that cannot be rendered correctly on the website)
- Rewording

### CODE REVIEW

File: wiki/Gameplay/Hit_object/Slider/Slidertail/zh.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ tags:
 
 # 滑条尾
 
-*滑条尾 (slidertail)*或**滑条终点 (sliderend)**指[滑条](/wiki/Gameplay/Hit_object/Slider)的终点位置。在控制光标跟随[s滑条体](/wiki/Gameplay/Hit_object/Slider/Sliderbody)运动后，玩家可以在此处松开按键。滑条尾只能在 [osu!](/wiki/Game_mode/osu!) [谱面](/wiki/Beatmap)中找到。取决于你正在使用的[皮肤](/wiki/Skin)版本，滑条尾可以是完全可见的一个圆圈，也可以成为滑条体不可见的一部分。滑条尾的颜色由谱师设置的连击颜色决定。
+*滑条尾 (slidertail)* 或 **滑条终点 (sliderend)** 指[滑条](/wiki/Gameplay/Hit_object/Slider)的终点位置。在控制光标跟随[滑条体](/wiki/Gameplay/Hit_object/Slider/Sliderbody)运动后，玩家可以在此处松开按键。滑条尾只能在 [osu!](/wiki/Game_mode/osu!) [谱面](/wiki/Beatmap)中找到。取决于你正在使用的[皮肤](/wiki/Skin)版本，滑条尾可以是完全可见的一个圆圈，也可以成为滑条体不可见的一部分。滑条尾的颜色由谱师设置的连击颜色决定。
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hit_object/Slider/Slidertail/zh.md:10]:
```suggestion
*滑条尾 (slidertail)* 或**滑条终点 (sliderend)** 指[滑条](/wiki/Gameplay/Hit_object/Slider)的终点位置。在控制光标跟随[滑条体](/wiki/Gameplay/Hit_object/Slider/Sliderbody)运动后，玩家可以在此处松开按键。滑条尾只能在 [osu!](/wiki/Game_mode/osu!) [谱面](/wiki/Beatmap)中找到。取决于你正在使用的[皮肤](/wiki/Skin)版本，滑条尾可以是完全可见的一个圆圈，也可以成为滑条体不可见的一部分。滑条尾的颜色由谱师设置的连击颜色决定。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Hit_object/zh.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ tags:
 
 # 打击物件
 
-*打击物件 (hit objects)*是游玩[谱面](/wiki/Beatmap)期间，玩家可以互动的元素。每个[游戏模式](/wiki/Game_mode)都拥有一组能够改变游戏体验的独特打击物件。对于每个游戏模式打击物件的概览，详见下表：
+*打击物件 (hit objects)* 是游玩[谱面](/wiki/Beatmap)期间，玩家可以互动的元素。每个[游戏模式](/wiki/Game_mode)都拥有一组能够改变游戏体验的独特打击物件。对于每个游戏模式打击物件的概览，详见下表：
```
Comment (YumeMuzi) [inline wiki/Gameplay/Hit_object/zh.md:9]:
```suggestion
*打击物件 (hit objects)* 是玩家游玩[谱面](/wiki/Beatmap)时，可以互动的元素。每个[游戏模式](/wiki/Game_mode)都拥有一组能够改变游戏体验的独特打击物件。对于各个模式打击物件的概览，详见下表：
```

Comment (YumeMuzi) [reply wiki/Gameplay/Hit_object/zh.md:9]: 加一下原文也有的 needs_cleanup 标签？

Comment (CloneWith) [reply wiki/Gameplay/Hit_object/zh.md:9]: 不知道需不需要，前面有的翻译文章也没加

Unsure whether to add the `need_cleanup` tag. Some  ZH translations didn't have it.

@TicClick for this

**Comment by TicClick** [comment]
> Unsure whether to add the need_cleanup tag. Some ZH translations didn't have it.
> @TicClick for this

the guidance is [as follows](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#articles-that-need-help): if a translation is messy or poorly translated, while the original article is fine, feel free to add `needs_cleanup: true`. if the original article has `needs_cleanup: true` in the first place, copy it over to the translation without a second thought.

**Review by YumeMuzi — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**